integration/src/integration/java/pl/allegro/tech/hermes/integration/PublishingAvroTest.java

```diff
@@ -28,10 +27,7 @@
 
 import static java.util.Collections.singletonMap;
 import static javax.ws.rs.core.MediaType.TEXT_PLAIN;
-import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
-import static javax.ws.rs.core.Response.Status.CREATED;
-import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
-import static javax.ws.rs.core.Response.Status.OK;
+import static javax.ws.rs.core.Response.Status.*;
```

Comment:
Minor: Please use direct imports instead of wildcards. In Hermes we have this unwritten rule that we use direct imports :)

Reply:
IDEA features.. ;)